Neos.Neos/Classes/Domain/Import/LiveWorkspaceIsEmptyProcessorFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Import;
+
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceFactoryDependencies;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceFactoryInterface;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceInterface;
+use Neos\ContentRepository\Export\Processors\EventExportProcessor;
+
+/**
+ * @implements ContentRepositoryServiceFactoryInterface<LiveWorkspaceIsEmptyProcessor>
+ */
+final readonly class LiveWorkspaceIsEmptyProcessorFactory implements ContentRepositoryServiceFactoryInterface
```

Comment:
except this we can avoid accessing this internal things ... probably good enough? But still against https://github.com/neos/neos-development-collection/pull/5286 :D

Reply:
maybe the logic could be part of the content repository or the `ContentStreamPruner` (eg rename that thing :D)